libraries/heft-config-file/src/ConfigurationFileBase.ts

```diff
@@ -624,7 +625,9 @@ export abstract class ConfigurationFileBase<TConfigurationFile, TExtraOptions ex
     projectFolderPath: string | undefined
   ): IConfigurationJson & TConfigurationFile {
     // Deep copy the configuration file because different callers might contextualize properties differently.
-    const result: IConfigurationJson & TConfigurationFile = structuredClone(entry.configurationFile);
+    const result: IConfigurationJson & TConfigurationFile = structuredClone<
```

Comment:
Add a `TODO` comment to replace this version of structuredClone with the built-in version once Node 16 support is dropped on the TikTok side